src/widgets/barchart/bar.rs

```diff
@@ -102,15 +122,28 @@ impl<'a> Bar<'a> {
 
     /// Set the text value printed in the bar.
     ///
+    /// `text_value` can be a [`&str`], `Number` or anything that can be converted into [`String`].
+    ///
     /// If `text_value` is not set, then the [`ToString`] representation of `value` will be shown on
     /// the bar.
     ///
+    /// examples
+    ///
+    /// from [`&str`] and [`String`].
+    /// ```
+    /// use ratatui::widgets::Bar;
+    ///
+    /// Bar::default().text_value("str");
+    ///
+    /// Bar::default().text_value(String::from("String"));
+    /// ```
+    ///
```

Comment:
```suggestion
    /// # Examples
    ///
    /// From [`&str`] and [`String`]:
    ///
    /// ```
    /// use ratatui::widgets::Bar;
    ///
    /// Bar::default().text_value("str");
    /// Bar::default().text_value(String::from("String"));
    /// ```
    ///
```